Integrations/JsonWhoIs/JsonWhoIs.py

```diff
@@ -0,0 +1,180 @@
+''' IMPORTS '''
+# std py packages
+
+# 3-rd party py packages
+import requests
+
+# local py packages
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+# Parameters received from the user
+PARAMS = demisto.params()
+
+# TOKEN
+TOKEN = PARAMS.get('token')
+
+# Service base URL
+BASE_URL = 'https://jsonwhois.com'
+
+# Headers to be sent in requests
+HEADERS = {
+    'Accept': 'application/json',
+    'Authorization': f'Token token={TOKEN}'
+}
+
+# Self signed certificate so no need to verify by default
+USE_SSL = not PARAMS.get('insecure', False)
+
+''' HELPER FUNCTIONS '''
+@logger
+def http_request(method, url_suffix, params=None, max_retry=3):
+    # A wrapper for requests lib to send our requests and handle requests and responses better
+    res = requests.request(
+        method,
+        BASE_URL + url_suffix,
+        verify=USE_SSL,
+        params=params,
+        headers=HEADERS
+    )
+    # Handle error responses gracefully
+    if res.status_code not in {200}:
+        if max_retry > 0:
+            max_retry -= 1
+            return http_request(method=method,
+                                url_suffix=url_suffix,
+                                params=params,
+                                max_retry=max_retry)
+
+        raise Exception(f'Error enrich url with JsonWhoIS API, status code {res.status_code}')
+    return res.json()
+
+
+def dict_by_ec(cur_dict: dict):
+    """ Create dict (Json) by entry contexts convention
+    Capitalize first char, remove nulls
+    :param cur_dict: dictionary
+    :return: dictionary by conventions
+    """
+    if not cur_dict:
+        return None
+    return {key.capitalize(): value for key, value in cur_dict.items() if cur_dict[key]}
+
+
+def list_by_ec(cur_list: list, needed_keys: list):
+    """ Create list of dict (Json) by entry contexts convention
+    Capitalize first char in dict, remove nulls, remove not needed parameters
+    :param cur_list: list of dict
+    :param needed_keys: key to save
+    :return: modified list by description above
+    """
+    if not cur_list:
+        return None
+    cur_list = [createContext(index, removeNull=True) for index in cur_list]
+
+    def cur_ec(index):
```

Comment:
index is a name of a numeric variable, very confusing...
try to find a better name